treeherder/perf/models.py

```diff
@@ -145,6 +162,25 @@ class PerformanceAlert(models.Model):
     t_value = models.FloatField(
         help_text="t value out of analysis indicating confidence "
         "that change is 'real'")
+    bug_number = models.PositiveIntegerField(null=True)
+
+    def save(self, *args, **kwargs):
+        # just a bit of extra business logic to make sure
+        # the status corresponds with other metadata
+        if self.revised_summary is not None:
+            self.status = PerformanceAlert.DUPLICATE
+        elif self.status == PerformanceAlert.DUPLICATE:
+            self.status == PerformanceAlert.UNTRIAGED
```

Comment:
In english: 

we're saying that if the item does not have a revised_summary, it can not be a duplicate. So reset it to untriaged, if it is marked as a duplicate. Contrariwise, if it *does* have a revised summary, it must be considered a duplicate.

Re: terminology. I didn't really think about it to be honest, but I think it's pretty common terminology both at Mozilla and in the greater world (just Yahoo it). I think it's ok to use.